crates/router/src/connector/powertranz.rs

```diff
@@ -87,6 +87,10 @@ impl ConnectorCommon for Powertranz {
         "application/json"
     }
 
+    fn get_currency_unit(&self) -> api::CurrencyUnit {
```

Comment:
Hi @Adithya4720 ,

This changes are not concerned with issue attached. Please refer #2052 for more info and also change the PR title accordingly.